### PR TITLE
Assistance msg

### PR DESCRIPTION
**Pull request by idegtiarov**
Add editable fields for messages when user is enrolled successfully  and unsuccessfully.

### CODE REVIEW

File: edxadapt/edxadapt.py [hunk 1]
```diff
@@ -60,7 +60,27 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.preferences
     )
 
-    editable_fields = ('params', 'skills', 'edx_adapt_api_url')
+    success_registration_msg = String(
+        default=_(
+            "Successfully registered in Edx Adapt, course is completely ready to start."
+        ),
+        display_name=_("Congrats with successful registeration"),
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: "Successful EdxAdapt registration message"

File: edxadapt/edxadapt.py [hunk 2]
```diff
@@ -60,7 +60,27 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
...
+        help=_("Congrats message student see after successfully enrolled in Edx Adapt"),
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: "Message displayed to student after successful registration in EdxAdapt."

File: edxadapt/edxadapt.py [hunk 3]
```diff
@@ -60,7 +60,27 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.preferences
     )
 
-    editable_fields = ('params', 'skills', 'edx_adapt_api_url')
+    success_registration_msg = String(
+        default=_(
+            "Successfully registered in Edx Adapt, course is completely ready to start."
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: "Your adaptive learning session is configured now. Please click Next button to start solve problems."

File: edxadapt/edxadapt.py [hunk 4]
```diff
@@ -60,7 +60,27 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
...
+        help=_("Congrats message student see after successfully enrolled in Edx Adapt"),
+        scope=Scope.content,
+    )
+
+    fail_registration_msg = String(
+        default=_(
+            "There was a technical issue while we were configuring your adaptive learning session. Please try to "
+            "refresh this page or contact technical staff if problem persists."
+        ),
+        display_name=_("Assistance message if enrollment fails"),
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: "Failed EdxAdapt registration message"

File: edxadapt/edxadapt.py [hunk 5]
```diff
@@ -102,7 +122,9 @@ def student_view(self, context=None):
         frag = Fragment(html.format(
             self=self,
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: `self` is not used in html, can be removed

File: edxadapt/edxadapt.py [hunk 6]
```diff
@@ -102,7 +122,9 @@ def student_view(self, context=None):
...
             anonymous_student_id=anonymous_student_id,
-            student_is_registered=self.student_is_registered
+            student_is_registered=self.student_is_registered,
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: `student_is_registered` is not used in html, can be removed

**Review by z4y4ts — CHANGES_REQUESTED** [review]

File: edxadapt/edxadapt.py (hunk 3 above)
Comment (idegtiarov) [reply edxadapt/edxadapt.py]: done

File: edxadapt/edxadapt.py (hunk 1 above)
Comment (idegtiarov) [reply edxadapt/edxadapt.py]: done

File: edxadapt/edxadapt.py (hunk 2 above)
Comment (idegtiarov) [reply edxadapt/edxadapt.py]: done

File: edxadapt/edxadapt.py (hunk 4 above)
Comment (idegtiarov) [reply edxadapt/edxadapt.py]: done

File: edxadapt/edxadapt.py (hunk 5 above)
Comment (idegtiarov) [reply edxadapt/edxadapt.py]: done

File: edxadapt/edxadapt.py (hunk 6 above)
Comment (idegtiarov) [reply edxadapt/edxadapt.py]: done

File: edxadapt/edxadapt.py [hunk 7]
```diff
@@ -100,16 +120,24 @@ def student_view(self, context=None):
...
-        frag.add_javascript(self.resource_string('static/js/src/edxadapt.js'))
+        frag.add_javascript(
+            html_parser.unescape(
+                Template(self.resource_string(
+                    'static/js/src/edxadapt.js'
+                )).render(Context({
+                    'anonymous_student_id': anonymous_student_id,
+                    'course_id': self.get_course_id(),
+                    'edx_adapt_api_url': self.edx_adapt_api_url.rstrip('/'),
+                    'params': self.params,
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: Hold on. I though we've moved `params` and `skills` into html template. Looks like merging went wrong.

Or do I miss something?

Comment (idegtiarov) [reply edxadapt/edxadapt.py]: That is weird, I am checking

File: edxadapt/edxadapt.py [hunk 8]
```diff
@@ -60,7 +60,27 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
...
+            "There was a technical issue while we were configuring your adaptive learning session. Please try to "
+            "refresh this page or contact technical staff if problem persists."
+        ),
+        display_name=_("Failed EdxAdapt registration message"),
+        help=_("Assistance message with steps student can do to improve Edx Adapt enrollment, e.g. email could be "
+               "added for asking for technical support."),
+        scope=Scope.content,
+    )
+
+    editable_fields = ('params', 'skills', 'edx_adapt_api_url', 'success_registration_msg', 'fail_registration_msg')
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py:83]: I guess it's about time to split this long line.

**Review by z4y4ts — APPROVED** [review]
You may want to make long lines comply with the code style. The rest looks good.

Make sure to squash it.

File: edxadapt/edxadapt.py (hunk 8 above)
Comment (idegtiarov) [reply edxadapt/edxadapt.py:83]: All line's length is less than 120 chars which is recommended by edx, so I suppose splitting is not needed.

**Pull request merged**